Few more fixes for Salt Bundle (#1480)
uyuni-project/uyuni-docs: modules/client-configuration/pages/contact-methods-saltbundle.adoc

> == Client Registration with Salt Bundle as a Minion

Author: @keichwa it's actually not a method.
It's better not to remove the Minion from this part. It could make some mess in the terms, but the Salt Clients could be different ones for example the minion connected with ZeroMQ or Salt SSH client which is not actually  the minion according to the terms of Salt, but this header is to the part describing the Minion only and this part makes no sense for Salt SSH or Salt client in general.
I would suggest the following:
```suggestion
== Client Registration with the Salt Bundle as a Minion
```

> == Client Registration with Salt Bundle as a Minion

Reviewer: I see your point.  In the past, they told us that we'd better avoid that term. I personally cannot judge about it --I'm neither a native speaker nor an expert in SF or where ever those strangers may appear `?-)`  Thus it might make perfect sense the way you say.

You can switch existing clients to the Salt Bundle method.
If you switch, the [package]``salt-minion`` package and its dependencies will stay installed.

> === Using the Salt Bundle with the Salt Minion

Author: Almost the same as the previous one. This part is exactly about using the Salt Bundle as the minion in the combination with original Salt Minion, but not about using the Salt Bundle by itself.

> === Using the Salt Bundle with the Salt Minion

Reviewer: `:check_mark:`

> The Salt Bundle can be used with the Salt Minion managed by the Salt Master other than {productname} Server at the same time.
> If the Salt Bundle is installed on a client {productname} Server will manage its configuration files, the configuration files of [literal]``salt-minion`` will not be managed in this case.
> For more information, see xref:client-configuration:registration-cli.adoc#_salt_bundle_configuration[Salt Bundle configuration].

Reviewer: ```suggestion
The Salt Bundle can be used as a client registration method even it the Salt Master runs on an own host, separately from the {productname} Server.
If the Salt Bundle is installed on a client {productname} Server will manage the client's configuration files, but the configuration files of [literal]``salt-minion`` will not be managed in this case.
For more information, see xref:client-configuration:registration-cli.adoc#_salt_bundle_configuration[Salt Bundle configuration].
```

Author: The Salt Bundle is not the registration method by itself. In your suggestion it's loosing the exact meaning of the. The second sentance not about all the client's configuration files, but exactly about Salt Bundle or Salt Minion config files.

Reviewer: ok, I see.  In this case we could consider to repeat the term "the Salt Bundle" or "the Client's Salt Bundle" instead of "its".  But this isn't that important.

Author: Fixed slightly different way with https://github.com/uyuni-project/uyuni-docs/pull/1480/commits/c350233476de0eb428eb32af2213a31797545398, is it fine now?

Reviewer: thanks, sounds good now!

[IMPORTANT]
====
To bootstrap a client with the Salt Minion managed by the Salt Master other than {productname} Server it is recommended to use [command]``mgr-bootstrap`` [option]``--force-bundle`` when generating the bootstrap script, or to set [option]``FORCE_VENV_SALT_MINION`` to `1` in the bootstrap script.

> For bootstrapping with {webui} [literal]``mgr_force_venv_salt_minion`` set to [literal]``true`` pillar can be specified globally. See xref:specialized-guides:salt/salt-states-and-pillars.adoc[Salt States and Pillars] for more information.

Reviewer: ```suggestion
For bootstrapping with {webui} [literal]``mgr_force_venv_salt_minion`` set to [literal]``true`` pillar can be specified globally.
For more information, see xref:specialized-guides:salt/salt-states-and-pillars.adoc[Salt States and Pillars].
```
